### DISCUSSION TITLE
How To Install MOV.AI Flow™?

### DISCUSSION THREAD

**Discussion by xsoulp** [MOV-AI/movai-flow#18] [Q&A] [answered]
I would like to know how to install MOVAI ?
Can someone post a tutorial?

**Comment by xsoulp** [comment, marked as answer]
please see the instructions at this link : https://github.com/MOV-AI/ce-project-studio/blob/main/README.md

**Reply by AlexFernandes-MOVAI** [reply]
Right, some documentation will also be available on readme.io

**Reply by xsoulp** [reply]
i am continuing discussions

**Reply by MithunKinarullathil** [reply]
It's available in the readme.io website now: https://flow.mov.ai/docs/installation
